mpv.py

```diff
@@ -479,6 +479,12 @@ def command(self, name, *args):
                 for arg in args if arg is not None ] + [None]
         _mpv_command(self.handle, (c_char_p*len(args))(*args))
 
+    def play(self):
```

Comment:
This name collides with ```play(self, filename``` defined [here](https://github.com/hatzel/python-mpv/blob/4bd99db1620933417f80ec2121c51e8408d4400d/mpv.py#L675). Maybe rename this to ```resume```?